src/components/forms/transaction/TransactionForm.tsx

```diff
@@ -150,7 +150,6 @@ async function onSubmit(data: FormValues, action: 'add' | 'update' | 'delete', o
 
   transaction.splits.forEach(split => {
     if (split.account.commodity.namespace !== 'CURRENCY') {
-      mutate('/api/investments', undefined);
```

Comment:
This is the main benefit, now this key is updated automatically by just calling the mutate below.